[0m[[0m[31merror[0m] [0m[0morg.apache.hadoop.mapred.FileAlreadyExistsException: Output directory file:/Users/jonathanhung/uic-code/cs441/homework1/src/main/resources/output/outdata.txt already exists[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.mapreduce.lib.output.FileOutputFormat.checkOutputSpecs(FileOutputFormat.java:164)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.mapreduce.JobSubmitter.checkSpecs(JobSubmitter.java:278)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.mapreduce.JobSubmitter.submitJobInternal(JobSubmitter.java:142)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.mapreduce.Job$11.run(Job.java:1677)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.mapreduce.Job$11.run(Job.java:1674)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.security.AccessController.doPrivileged(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/javax.security.auth.Subject.doAs(Subject.java:423)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1953)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.mapreduce.Job.submit(Job.java:1674)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.mapreduce.Job.waitForCompletion(Job.java:1695)[0m
[0m[[0m[31merror[0m] [0m[0m	at Word2VecDriver$.main(MapWord2Vec.scala:162)[0m
[0m[[0m[31merror[0m] [0m[0m	at Word2VecDriver.main(MapWord2Vec.scala)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.reflect.Method.invoke(Method.java:566)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.invokeMain(Run.scala:144)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.execute$1(Run.scala:94)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.$anonfun$runWithLoader$5(Run.scala:121)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run$.executeSuccess(Run.scala:187)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.runWithLoader(Run.scala:121)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$bgRunTask$6(Defaults.scala:2038)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$termWrapper$2(Defaults.scala:1977)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.util.Try$.apply(Try.scala:213)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.internal.BackgroundThreadPool$BackgroundRunnable.run(DefaultBackgroundJobService.scala:367)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.Thread.run(Thread.java:834)[0m
[0m[[0m[31merror[0m] [0m[0m(Compile / [31mrun[0m) org.apache.hadoop.mapred.FileAlreadyExistsException: Output directory file:/Users/jonathanhung/uic-code/cs441/homework1/src/main/resources/output/outdata.txt already exists[0m
